tests/lib/test_test_suite.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+from uuid import uuid4
+from ...src.vellum.lib.test_suite import VellumTestSuite
+import pytest
+from requests_mock import Mocker as RequestsMocker
+
+
+# Get started with writing tests with pytest at https://docs.pytest.org
+def test_vellum_test_suite__external__basic(requests_mock: RequestsMocker) -> None:
+    """Verify that the Vellum test suite could execute on External executions."""
+
+    # GIVEN an external execution
+    def mock_execute_thing(input_a: str, input_b: dict[str, Any]) -> dict[str, Any]:
+        """This could be the invocation of a Prompt, Langchain chain, etc."""
+
+        return {"output_a": "Example string output", "output_b": {"key": "value"}}
```

Comment:
Doesn't have to be in V1, but I think that for this executable function, we should support this input/output interface, but should also support:

```
def mock_execute_thing(inputs: list[CompiledVellumVariableValue]) -> list[CompiledVellumVariableValue]:
        pass
```